scripts/annotation.js

```diff
@@ -154,9 +139,45 @@ $(document).ready( ()=> {
                 seadragon.hide();
                 image_div.show();
             }
-            image_div.attr('src', '/images?id=' + image);
+            let imgURL = '/images?id=' + image;
+            $.get(imgURL)
+                .done(() => {
+                    // knowing the image exists
+                    image_div.attr('src', imgURL);
```

Comment:
the $.get call is uneccessary here, you're changing the src of the image div to the img url, which will automatically download the picture